saleor/shipping/models.py

```diff
@@ -173,6 +180,17 @@ def __repr__(self):
         )
 
 
+class ShippingMethodZipCode(models.Model):
+    shipping_method = models.ForeignKey(
+        ShippingMethod, on_delete=models.PROTECT, related_name="zip_codes"
+    )
+    start = models.CharField(max_length=32)
+    end = models.CharField(max_length=32)
```

Comment:
The value which you get for the `end` can be blank/null. Not sure if the field will be able to handle it.